src/common/Backend.ts

```diff
@@ -266,7 +266,8 @@ export interface BackendEventMap {
     };
     'backend-status': {
         message: string;
-        percent: number;
+        progress: number;
+        statusProgress?: number;
```

Comment:
This type is still wrong.
```suggestion
        statusProgress?: number | null;
```